### PR TITLE
Add push-to-talk sound effects

### PR DESCRIPTION
**Pull request by dbkr**
With some hastily made placeholder sounds. The 'blocked' sound is probably a bit over-aggressive but they're placeholders so I'll let it go and leave it to an actual sound designer!

Fixes https://github.com/vector-im/element-call/issues/296

### CODE REVIEW

File: src/room/usePTT.ts [hunk 1]
```diff
@@ -147,8 +188,8 @@ export const usePTT = (
     }
 
     window.addEventListener("keydown", onKeyDown);
-    window.addEventListener("keyup", onKeyUp);
-    window.addEventListener("blur", onBlur);
+    //window.addEventListener("keyup", onKeyUp);
```
Comment (robertlong) [inline src/room/usePTT.ts:191]: Are these meant to be left commented out? Releasing spacebar no longer works without this.

**Review by robertlong — CHANGES_REQUESTED** [review]
Yeah the call blocked sound effect is pretty aggressive 😅 but the sound effects seem to work alright. However I noticed a few lines are commented out that break the PTT functionality.

File: src/room/usePTT.ts (hunk 1 above)
Comment (dbkr) [reply src/room/usePTT.ts]: Oops - good catch. This is how I'd been testing the admin talkover without two physical computers, but forgot to put it back afterwards. 🤦

**Comment by dbkr** [comment]
Assuming this is good now & merging.

**Pull request merged**